### PR TITLE
feat: use three color in api

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**
#### Type of change
Refactor 

#### Jira ticket :blue_book:

https://cognitedata.atlassian.net/browse/REV-542

## Description :pencil:

Replaces tuples of RGB values by THREE.Color instances in the API.

## How has this been tested? :mag:

Saw through all documentation pages using styling (there are a lot) and the example.

## Checklist :ballot_box_with_check:

- [x] I am proud of this feature.
- [x] I have performed a self-review of my own code.
- [x] I have added PR type (Feat, Bug, Chore, Test, Docs, Style, Refactor) to the PR title.
- [x] I have manually tested this for different scenarios (different models, formats, devices, browsers).
- [x] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the public documentation.
- [x] I have added unit and visuals tests to prove that my feature works to the best of my ability.
- [x] I have refactored the code for readability to the best of my ability.
- [x] I have checked that my changes do not introduce regressions in the public documentation.
- [x] I have outlined any known defects / lacking capabilities in the contents of this PR.
- [x] I have listed any remaining work that should follow this PR in the description or jira/miro/etc.
- [x] I have added TSDoc to any public facing changes.
- [x] I have added "developer documentation" in any package README.md that is related / applicable to this PR.
- [x] I have noted down and am currently tracking any technical debt introduced in this PR.
- [x] I have thoroughly thought about the architecture of this implementation.
- [ ] I have asked peers to test this feature.

### CODE REVIEW

File: examples/src/utils/NodeStylingUI.ts [hunk 1]
```diff
@@ -126,6 +126,7 @@ export class NodeStylingUI {
     };
     function updateState() {
       state.color = colorToHexString(appearance.color);
+      console.log('State color = ', state.color);
```
Comment (christjt) [inline examples/src/utils/NodeStylingUI.ts:129]: Should this be removed?

File: viewer/packages/pointcloud-styling/src/PointCloudAppearance.ts [hunk 2]
```diff
@@ -2,15 +2,17 @@
...
-  color?: [number, number, number];
+  color?: Color;
   visible?: boolean;
 };
 
 export type CompletePointCloudAppearance = Required<PointCloudAppearance>;
 
 export const DefaultPointCloudAppearance: CompletePointCloudAppearance = {
-  color: [0, 0, 0],
+  color: new Color(0, 0, 0),
```
Comment (christjt) [inline viewer/packages/pointcloud-styling/src/PointCloudAppearance.ts:15]: Think default is (0, 0, 0) ?

**Review by christjt — APPROVED** [review]
LGTM! Only a couple small nits.

File: examples/src/utils/NodeStylingUI.ts (hunk 1 above)
Comment (haakonflatval-cognite) [reply examples/src/utils/NodeStylingUI.ts:129]: :eyes: Oh wow

**Pull request merged**